src/30_Advanced/Image_Galleries_with_amp-carousel.html

```diff
@@ -75,6 +75,37 @@
       border-style: solid;
     }
   </style>
+  <!-- ## Metadata
```

Comment:
Can we move this to the end of the sample and put the metadata into a code block? I'd prefer to avoid having invalid metadata in the resulting sample file.